src/FSharp.Core/list.fsi

```diff
@@ -1245,6 +1398,9 @@ module List =
     [<CompiledName("Last")>]
     val last: list:'T list -> 'T
 
+    /// 
+    /// Time complexity: O(n) - where n is the list length
+    /// Space complexity: O(1)
```

Comment:
The added complexity information should be within the `<summary>` tag, right before the closing tag.